slatedb-bencher/src/args.rs

```diff
@@ -64,6 +64,9 @@ pub(crate) struct DbArgs {
 
     #[arg(long, help = "The size in bytes of the block cache.")]
     pub(crate) block_cache_size: Option<u64>,
+
+    #[arg(long, help = "The size in bytes of the meta cache.")]
+    pub(crate) meta_cache_size: Option<u64>,
```

Comment:
Can you add an option for toggling the type of cache so we can bench with either a split or unified cache?